index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>portfolio mobile</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&display=swap" rel="stylesheet"> 
+    <link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@300&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+    <script src="https://kit.fontawesome.com/ee00ee8a15.js" crossorigin="anonymous"></script>
+</head>
+<body>
+    <header class="header">
+        <nav class="name">
+            <a href="#">John Doe</a>
+            <a href="#"><img src="img/Union.png" alt="hamburger"></a>
+        </nav>
```

Comment:
### OPTIONAL 
- [ ] Kindly remove the logo from the nav tag only the hamburger icon should be in the nav tag,  because this is part of the project requirements👍🏿